src/main/resources/db/migration/V0_0_40__nc_event_add_name_and_description_cols.sql

```diff
@@ -0,0 +1,2 @@
+alter table nc_event add column name VARCHAR(50);
```

Comment:
This might have confuse you (with regards to recipients). Here we are extending NC based on OSK requirement. But here I think we can consider name and description of event to be a generic think. No matter what payload, external system can (doesn't have to) provide name and description for that event which can be used also in the NC UI.
So this is ok, in my opinion